tensorflow/contrib/rnn/python/ops/rnn_cell.py

```diff
@@ -1330,6 +1330,153 @@ def assert_shape_match(inp, out):
     return (res_outputs, new_state)
 
 
+class LayerNormBasicGRUCell(rnn_cell_impl._LayerRNNCell):
```

Comment:
Does it make sense to have a layer norm wrapper, just like we have a dropout wrapper?

Reply:
Hello @drpngx,
Thanks for the review. I agree a wrapper would be ideal, but unlike dropout here layer normalization changes gates equations so it seems harder to implement it with a wrapper. Also, the way it is applied is not consistent between RNN cells.

Reply:
I'm not sure that this is correct. Are you sure?

Reply:
I mean most of the time layer normalization is applied between linear projection and non linearity, but for example in the case of LSTM it is also applied on the internal state c when computing the hidden state h.